File: AuctionApplication-UI/src/app/models/Bid.ts
export class Bid {
    auctionItemId: number
    maxAutoBidAmount: number
    bidderName: String

    constructor(auctionItemId: number, maxAutoBidAmount: number, bidderName: String) {
        this.auctionItemId = auctionItemId;
        this.maxAutoBidAmount = maxAutoBidAmount;
        this.bidderName = bidderName;
    }
}

File: AuctionApplication/src/main/java/com/mhoveidafar/demo/domain/Bid.java
package com.mhoveidafar.demo.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Bid {

    private int auctionItemId;

    private double maxAutoBidAmount;

    private String bidderName;
}
